src/ResourceManager/DevSpaces/Commands.DevSpaces/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="AutoMapper" version="6.2.1" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DevSpaces" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
Remove clientruntime, clientruntime.azure, and system.management.automation - all are included in the common dependencies.

Reply:
Done